tests/test-sources/plugins/by-name/copilot-chat/default.nix

```diff
@@ -1,70 +1,31 @@
 {
 
   empty = {
-    plugins.copilot-chat.enable = true;
+    plugins.copilot-chat.enable = false;
```

Comment:
?

Reply:
I assume we are disabling a broken test that is too much effort to fix? If so, that should be clear in a comment and in the commit message.

Ideally, I'd prefer we scale back the test until it covers things that we feel confident maintaining options for.

I'm also concerned that if we had these tests, maybe we also declare/document corresponding options? If so we should look at removing any old/dropped options from the plugin.